### PR TITLE
Bala's changes to SCIM guide

### PR DESCRIPTION
**Pull request by mystiberry-okta**
Not ready for review, needs the flowchart added and developers@okta.com added to the "our team" (JF okayed).

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@balathiruvalluvan-okta  Just a reminder to send me the flowchart.

File: _source/_standards/SCIM/index.md [hunk 1]
```diff
@@ -820,32 +862,85 @@ Check your Profile Attributes as follows:
         application appears to support attributes that are not
         supported by your SCIM API.
 
-        You can delete an attribute by selecting an attribute, then
+        You can delete an attribute by selecting it, then
         clicking the "Delete" button located in right hand attribute details pane.
+        Before removing, check the mapping between Okta and Application and
+        **remove they mappings** forthe attribute(s) to be deleted.
```
Comment (balathiruvalluvan-okta) [inline _source/_standards/SCIM/index.md]: remove the* mappings, not sure if its a typo @mystiberry-okta

File: _source/_standards/SCIM/index.md [hunk 2]
```diff
@@ -820,32 +862,85 @@ Check your Profile Attributes as follows:
...
-        -   Display name
-        -   Variable name
-        -   External name
-        -   External namespace
-        -   Data type
+        -   Display name -- Example: Family name
+        -   Variable name -- Example: isvapp.familyName
+        -   External name -- Example: name.familyName
+        -   External namespace -- Example: urn:ietf:params:scim:schemas:core:2.0:User
+        -   Data type -- Example: String
```
Comment (balathiruvalluvan-okta) [inline _source/_standards/SCIM/index.md]: @mystiberry-okta i think i forgot to mention that i also added a new screenshot for the this. Please upload that under this section. I have also mentioned this in the wiki

Comment (mystiberry-okta) [inline _source/_standards/SCIM/index.md]: Added!

File: _source/_standards/SCIM/index.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_standards/SCIM/index.md]: sorry about that! also fixed "forthe"

**Comment by balathiruvalluvan-okta** [comment]
Looking good, we are good to merge

**Comment by mystiberry-okta** [comment]
Thumbs up from Bala in hipchat. yay!

**Pull request merged**